staging/src/k8s.io/dynamic-resource-allocation/kubeletplugin/draplugin.go

```diff
@@ -407,7 +410,7 @@ func (d *draPlugin) Stop() {
 
 // PublishResources implements [DRAPlugin.PublishResources]. Returns en error if
 // kubeClient or nodeName are unset.
-func (d *draPlugin) PublishResources(ctx context.Context, resources Resources) error {
+func (d *draPlugin) PublishResources(_ context.Context, resources resourceslice.DriverResources) error {
```

Comment:
Why do we pass a context at all if it is never used?

Reply:
Indeed, `PublishResources` is asynchronous update and return immediately. It seems that there is no need to use ctx to set a timeout or cancel the call. In addition, draPlugin itself also has backgroundCtx, which should be able to be deleted.  @pohly

Reply:
Because it's more future-proof. If at some point we do need a context (for example, for contextual logging), then adding it would be another API break.

Reply:
got this.